mlrun/datastore/azure_blob.py

```diff
@@ -135,3 +136,69 @@ def listdir(self, key):
     def rm(self, path, recursive=False, maxdepth=None):
         path = self._convert_key_to_remote_path(key=path)
         super().rm(path=path, recursive=recursive, maxdepth=maxdepth)
+
+    def get_spark_options(self):
+        res = {}
+        st = self.get_storage_options()
+        service = "blob"
```

Comment:
I think it's fine to inline. Not like we're going to add more services.

Reply:
you never know . I would leave it